GlotovES/HW7/task_3.py

```diff
@@ -0,0 +1,11 @@
+def calculate_character_count(input_string):
```

Comment:
good job
look at this version
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result